cache/cache.go

```diff
@@ -136,6 +141,46 @@ func collectIncludePaths(homeDir, projectDir string, cacheLevel cache.Level) ([]
 	return includePths, nil
 }
 
+/*
+If the indicator is version controlled in git and has changes, we create a copy of it with its original content.
+*/
+func prepareUnmodifiedIndicator(indicator string) string {
```

Comment:
would be better to return the errors, and handle them (print/omit) on the caller side

Reply:
Giving error is not a contract of this method. I don't see why would it be better to handle error cases in the caller method than here. We don't need to propagate errors any further